votor/src/certificate_pool_service.rs

```diff
@@ -276,6 +304,7 @@ impl CertificatePoolService {
             return;
         }
         *highest_parent_ready = new_highest_parent_ready;
+        CertificatePoolServiceStats::incr_u16(&mut stats.new_parent_ready);
```

Comment:
we should increment this on line 294, right now it will only notify on new highest parent ready.